Scripts/parameters.py

```diff
@@ -2879,6 +2879,18 @@
         ("age_65-99", "male"): 1.8737,
     },
 }
+car_density = {
+    "constant": 0.0,
+    "generation": {
+        "share_detached_houses": 2.408e-01, # Originally estimated for percentages (0-100), here transformed
```

Comment:
Zone data input values for "share_detached_houses" are in range 0-100 (scenario input data in Sharepoint). Is this correct or should the values be between 0-1? I think this might cause the overflow in demand matrices.

Reply:
Yes, I think the values should be 0-1

Reply:
Yes, I think the values should be 0-1